config.mak.uname

```diff
@@ -697,18 +697,10 @@ else
 		ifneq (,$(findstring -O,$(filter-out -O0 -Og,$(CFLAGS))))
 			BASIC_LDFLAGS += -Wl,--dynamicbase
 		endif
-		ifeq (MINGW32,$(MSYSTEM))
-			prefix = /mingw32
-			HOST_CPU = i686
-			BASIC_LDFLAGS += -Wl,--pic-executable,-e,_mainCRTStartup
-		endif
 		ifeq (MINGW64,$(MSYSTEM))
 			prefix = /mingw64
 			HOST_CPU = x86_64
 			BASIC_LDFLAGS += -Wl,--pic-executable,-e,mainCRTStartup
-		else
-			COMPAT_CFLAGS += -D_USE_32BIT_TIME_T
-			BASIC_LDFLAGS += -Wl,--large-address-aware
```

Comment:
Again, this is the 32-bit support that is removed. We're not there yet. I vaguely remember that we'll need to support 32-bit builds in Git for Windows until 2025, so this is waaaaay too early.